package baidu.bfs.master_slave;

option cc_generic_services = true;

message AppendLogRequest {
    optional bytes log_data = 1;
    optional int32 offset = 2;
}

message AppendLogResponse {
    optional bool success = 1;
    optional bool offset = 2;
}

service MasterSlave {
    rpc AppendLog(AppendLogRequest) returns(AppendLogResponse);
}
